clients/p4rt-ctl-cpp/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+# CMake build file for p4rt_perf_test
+#
+# Copyright 2023-2024 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+#
+
+include(CheckLibraryExists)
+
+set(PROTO_INCLUDES
+    # Protobuf C++ header files.
+    ${PB_OUT_DIR}
+)
+
+#################
+# p4rt_perf_test #
+#################
+
+add_library(p4rt_perf_test_o OBJECT
+    p4rt_perf.cc
+    p4rt_perf_session.cc
+    p4rt_perf_session.h
+    p4rt_perf_tls_credentials.cc
+    p4rt_perf_tls_credentials.h
+    p4rt_perf_simple_l2_demo.cc
+    p4rt_perf_util.cc
+)
+
+target_compile_options(p4rt_perf_test_o PRIVATE -O3)
+target_include_directories(p4rt_perf_test_o PRIVATE
```

Comment:
Why is this here? As a rule, you only need to specify a directory in the binary tree if you need to include a generated header file.

Reply:
cleaned up CMakeList.txt.